Tests/lib/taskRunner.ts

```diff
@@ -49,10 +50,15 @@ export class TaskRunner extends events.EventEmitter {
 	private _taskPath: string;
 	private _tempPath: string;
 	private _commands: string[];  
+	private _ignoreSlashes: boolean;
 
 	public ran(cmdLine: string): boolean {
 		var executed: boolean = false;
 		this._commands.forEach((cmd: string)=>{
+			if (this._ignoreSlashes) {
```

Comment:
It looks like we're not ignoring slashes, just replacing back ones with forward ones...